components/footer/index.js

```diff
@@ -114,7 +115,7 @@ const SiteLinksContainer = styled.div`
 const Footer = (props) => (
   <FooterContainer {...props}>
     <FlexSection>
-      <Text>Interested in Taking Control of your Career?</Text>
+      <Text>Let’s get started</Text>
       <StyledApplyLink />
```

Comment:
Related to my other comment, might need to add the `text` prop here instead of changing the default text value.

Reply:
+1, we should leave the default to where it's used in 2 places ("Join the waitlist") and pass in "Apply to Join Us" to override.